openwisp-utils-qa-checks

```diff
@@ -98,7 +98,7 @@ runcheckcommit() {
 
 runcheckpendingmigrations() {
   if [ ! -f "./tests/manage.py" ]; then
-    echo "File manage.py not found\nSkipping Make Migration Check"
```

Comment:
we would still need a new line at the end of the string (`echo` automatically adds a new line at the end, `printf` doesn't).

What about simply use the following?

```shell
echo "File manage.py not found, skipping Make Migration Check."
```